src/AssetsPage/index.jsx

```diff
@@ -16,13 +16,18 @@ class AssetsPage extends React.Component {
   }
 
   componentDidMount() {
-    this.props.getAssets(this.props.assetsParameters);
+    this.props.getAssets(this.props.assetsParameters, this.props.courseDetails);
   }
 
   componentDidUpdate(prevProps) {
+    if (prevProps.courseDetails !== this.props.courseDetails) {
+      this.props.getAssets(this.props.assetsParameters, this.props.courseDetails);
+    }
+
     if (prevProps.assetsParameters !== this.props.assetsParameters) {
+      // if filters changed, update the assetsList
       // TODO: consider using the reselect library for this
-      this.props.getAssets(this.props.assetsParameters);
+      this.props.getAssets(this.props.assetsParameters, this.props.courseDetails);
     }
```

Comment:
Reading back through this, I could see these two combining to be an `||`, I can't remember at the moment if there was a reason for not doing that intentionally.  Can anyone think of a reason why combining these wouldn't be ideal?

Reply:
👍 cool I'll `||` these two comparisons then